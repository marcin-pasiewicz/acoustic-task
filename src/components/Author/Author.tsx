import React, { ReactElement } from 'react'

interface Props {
  value: string;
}

export default function Author({ value }: Props): ReactElement {
  return (
    <div>
      {value}
    </div>
  )
}
