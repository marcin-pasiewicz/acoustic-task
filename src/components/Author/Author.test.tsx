import React from 'react';
import { render } from '@testing-library/react';
import Author from './Author';

describe('<Author/>', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Author value="Joe Doe" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
