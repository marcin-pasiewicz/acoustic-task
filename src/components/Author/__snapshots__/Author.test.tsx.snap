// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Author/> renders correctly 1`] = `
<DocumentFragment>
  <div>
    Joe Doe
  </div>
</DocumentFragment>
`;
